thanosql/util.py

```diff
@@ -115,3 +117,13 @@ def print_video(df, print_option):
         print(video_full_path)
         display(Video(video_full_path, embed=True))
     return
+
+def stream_sql_results(conn: Connection, query_string: str) -> pd.DataFrame:
+    dfs = []
+    for chunk_df in pd.read_sql_query(
+        text(query_string), 
+        conn.execution_options(stream_results=True),
```

Comment:
일단.. for문 밖에서 conn = conn.execution_options(stream_results=True) 하고
```suggestion
        conn=conn,
```
이렇게 해주시고..

그나저나 with engine.connect() as conn: 여기에서 stream_results=True를 주면 안되나요?

Reply:
DROP query 실행시키는 conn.execute가 있어서 streaming 할때에만 적용 싶었습니다!

Reply:
@stevensjcho 
stream 옵션 주면 drop 쿼리가 아예 에러나나요 혹시?

별차이 없으면 scope 넓혀서 stream 줘버리죠.. 이렇게 두면 나중에 코드 유지 보수할때 stream 옵션 빼먹고 쓸 확률이 높아져서요ㅠ

Reply:
넵 stream options 넣는 걸로 바꿨습니다.